src/angularjs/src/app/places/list/places-list.controller.js

```diff
@@ -34,13 +41,40 @@
             ctl.getPrev = getPrev;
             ctl.places = [];
 
-            ctl.filters = {};
+            ctl.neighborhoodFilter = null;
+
+            ctl.sortBy = sortingOptions[0]; // default to alphabetical order
+            ctl.sortingOptions = sortingOptions;
+
+            ctl.getPlaces = getPlaces;
 
             getPlaces();
+            loadOptions();
+            $scope.$watch(function(){return ctl.neighborhoodFilter;}, filterNeighborhood);
```

Comment:
This watcher should probably be manually cleaned up in the `$scope.destroy` method